### DISCUSSION TITLE
对不起，请问在这里的zip格式安装包，要怎么安装，教程链接也好，真的找不着

### DISCUSSION THREAD

**Discussion by nanbeiwap** [the1812/Bilibili-Evolved#2925] [提问 / Q&A] [answered]
或者告诉我点关键词，我自己搜也行，真的很想安装这个软件啊

**Comment by Dobby233Liu** [comment, marked as answer]
https://github.com/the1812/Bilibili-Evolved#安装

**Reply by nanbeiwap** [reply]
好的好的，万分感谢

**Reply by nanbeiwap** [reply]
兄弟，网址里面没有教程啊

**Reply by Geeyun-JY3** [reply]
网址指向的就是本项目 README，写得很清楚，能不能先认真看完指向的这一节，也就几百字。

**Reply by nanbeiwap** [reply]
感谢两位的回复，还是看不懂，不过很感谢，，，

**Reply by Geeyun-JY3** [reply]
装了脚本管理器后，在下方表格选择想安装的版本，点击相应的 `安装` 进入油猴脚本的安装页面，再点一次 <kbd>安装</kbd> 就完事了。。。。。

**Reply by nanbeiwap** [reply]
啊，兄弟，我才看懂你俩的回复，，是我没说清楚，，其实想问的是类似github上面有的zip文件，是要怎么安装，是要想办法打包成exe还是要借助什么工具，.....（油猴安装脚本，其实还是会的）

**Reply by the1812** [reply]
那个 zip 是源代码自动打包, 没啥用的...

**Reply by nanbeiwap** [reply]
那如果下载列表里没有exe文件可以下载，还能去哪里安装吗，，，或者有的安装过了，但是出新版本的压缩包了，要怎么才能更新呢

**Reply by Geeyun-JY3** [reply]
这是油猴脚本啊，要 exe 干啥。。

**Reply by nanbeiwap** [reply]
这个网站里面有挺多项目是有挺多zip的那种的，xxxxx_win-x64.zip这种的，下下来不会安，，，看到这个也是下载的zip文件，我以为这个也要这样安装呢，，

**Reply by Geeyun-JY3** [reply]
GitHub 上啥乱七八糟的项目都有，就算是桌面端应用，也不是每个项目都会往 Release 放安装包的。

顺带提醒一句，你说的这种应该是 Windows 对应位数的便携版应用压缩包，解压就能用。

**Reply by nanbeiwap** [reply]
嗯嗯，有的好像不行，不过算了，不纠结了，，谢谢你，告诉我这么多
